### ISSUE TITLE
Use this settings with shadowsocks-libev

### ISSUE THREAD

**Issue by monochrome-yeh** [hwdsl2/setup-ipsec-vpn#59]
Hello,

I've been using this vpn, it's works fine.

however I would like to build shadowsocks-libev in the same VPS,

it doesn't work for me, I think that has iptable settings problem.

therefore, how could I adjust iptable that would be work for shadowsocks-server ?

FYI:

`local port`: 1080
`server port`: 8898

**Comment by hwdsl2** [comment]
@monochrome-yeh If using Ubuntu/Debian, edit /etc/iptablee.rules and add rules for SS:

```
-A INPUT -p tcp --dport 8898 -j ACCEPT
-A INPUT -p udp --dport 8898 -j ACCEPT
```

Note: The above lines should be placed immediately below `-A INPUT -p tcp --dport 22 -j ACCEPT`.

Save the iptables.rules file and reboot your server.

**Comment by monochrome-yeh** [comment]
OIC, Thanks,
But I've did it before, it's wired.
Facebook web site could render on browser,
But the pictures has network problem.

I dont know why...

**Comment by monochrome-yeh** [comment]
I'd used another vps to built ssserver, it's also works,
Indeed, I haven't been using the iptable for that.

**Comment by hwdsl2** [comment]
@monochrome-yeh Not sure about that. Try searching the Shadowsocks community or raise an issue in that repo. Does not look like an IPTables or VPN related problem. Also you may try flushing the DNS cache on your PC: Open an elevated command prompt and run ipconfig /flushdns. Then reboot. You may try a different browser as well.

**Comment by monochrome-yeh** [comment]
Oops, you closed too fast.
I said I'd built ssserver without iptable on another bps before,
It's works for me.

**Comment by hwdsl2** [comment]
@monochrome-yeh It looks like I misread your response. Reopening for further troubleshooting. Did you try clearing your DNS cache or use a different browser? Any other details you could share regarding this issue?

**Comment by hwdsl2** [comment]
@monochrome-yeh Hello again! I tested your use case on my own VPS, and cannot reproduce the issue you encountered. After setting up shadowsocks-libev and then install the VPN, I modified IPTables rules as mentioned in my comment above. Finally I tested connections via the shadowsocks-windows client and the IPsec/L2TP VPN client in Windows, respectively. In both cases I am able to browse FB with no problem at all.

Closing this issue as I am unable to reproduce it. It is probably unrelated to either IPTables or the VPN.

**Comment by monochrome-yeh** [comment]
Sorry, I'm working at the moment,
so I can't reply you quickly.

I'm using [shadowsocks-libev](https://github.com/shadowsocks/shadowsocks-libev) and start with udp replay mode.

I've encountered this issue, when I installed **setup-ipsec-vpn**.

I couldn't show you my iptable settings now, because I didn't take my computer go out,
but I will show you later this night.

I think my problem it's how to forward whole the tcp/udp package with **setup-ipsec-vpn** 's iptable settings.

FYI: Kindly use `-u` parameter to start the experiment.
 - https://github.com/shadowsocks/shadowsocks-libev#advanced-usage (I'm confused in this part when I want to mix both of iptables.)
